### PR TITLE
refactor(SettingsProvider): code

### LINKED ISSUE #393: settings-providers

### PR DESCRIPTION
**Pull request by samandarkuchkarov**
Fixes #393 

## Proposed Changes

  -Change the imports to absolute:
  -Change the colors from the constants to similar ones in Color
Simulator Screen Shot - iPhone 14 - 2022-12-08 at 16 01 11

## Checklist

- [x] I have read the rules of [**contribution**](https://github.com/haqq-network/haqq-wallet/blob/main/docs/contribution.md#important-steps).

### CODE REVIEW

File: src/components/ui/data-content.tsx [hunk 1]
```diff
@@ -3,10 +3,9 @@ import React from 'react';
 import {StyleSheet, View, ViewStyle} from 'react-native';
 
 import {Color} from '@app/colors';
+import {Text} from '@app/components/ui/text';
```
Comment (gHashTag) [inline src/components/ui/data-content.tsx:6]:
```suggestion
import {Text} from '@app/components/ui';
```

**Review by gHashTag — CHANGES_REQUESTED** [review]
Pls accept suggestion

**Review by gHashTag — APPROVED** [review]

**Review by vivalaakam — APPROVED** [review]

**Review by gHashTag — APPROVED** [review]

**Review by vivalaakam — CHANGES_REQUESTED** [review]
revert [ios/haqq/haqq.entitlements](https://github.com/haqq-network/haqq-wallet/pull/577/files#diff-d277b486442c6aa3b3a287d2d3d118dd187b9eb1325321010173170cc4f7904e)

File: src/screens/.eslintignore [hunk 2]
```diff
@@ -0,0 +1 @@
+haqq.entitlements
```
Comment (vivalaakam) [inline src/screens/.eslintignore:1]: Why it there? Enough in the root

Comment (samandarkuchkarov) [reply src/screens/.eslintignore:1]: ok removing

**Comment by vivalaakam** [comment]
Are you sure that you remove right file?)

**Review by vivalaakam — APPROVED** [review]

**Pull request merged**